eng/restore/netcoreapp/netcoreapp.depproj

```diff
@@ -3,32 +3,18 @@
        With this, we are able to compile assemblies against shipped, stable versions of .NET Core. -->
   <PropertyGroup>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
-    <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.7</NETCoreAppPackageVersion>
-    <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">2.1.3</NETCoreAppPackageVersion>
-    <!-- These needs to be replaced once we have a RTM -->      
-    <RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp3.0'">3.0.0-preview7-27912-14</RuntimeFrameworkVersion>
-    <DisableImplicitFrameworkReferences Condition="'$(TargetFramework)' == 'netcoreapp3.0'">false</DisableImplicitFrameworkReferences>
+    <NetCoreAppVersion Condition="$(TargetFramework.StartsWith('netcoreapp'))">$(TargetFramework.SubString(10))</NetCoreAppVersion>
+    <IsNetCoreApp2x Condition="$(NetCoreAppVersion.StartsWith('2.'))">true</IsNetCoreApp2x>
```

Comment:
Is `NetCoreAppVersion` really needed? We could just do, 

```xml
<IsNetCoreApp2x Condition="$(TargetFramework.StartsWith('netcoreapp2.'))">true</IsNetCoreApp2x>
```

Reply:
less code, good suggestion.